docs/api/rbac/rbac-example.mdx

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_position: 2
+title: Create RBAC policy
+---
+
```

Comment:
Add here:

"At our company, have a shared todo app that allows users to create, read, update, toggle and delete tasks. Let's see how it would be created in practice with the use of the RBAC API" and delete it below.